wpilibc/src/main/native/include/frc/util/Color.h

```diff
@@ -808,6 +808,15 @@ class Color {
     }
   }
 
+  /**
+   * Return this color represented as a hex string.
+   *
+   * @return a string of the format <tt>\#RRGGBB</tt>
+   */
+  std::string_view HexString() const {
+    return fmt::format("#{:02X}{:02X}{:02X}", static_cast<int>(255.0 * red), static_cast<int>(255.0 * green), static_cast<int>(255.0 * blue));
```

Comment:
This is returning a string_view of a temporary std::string.  Return a std::string instead.  Also, to avoid a fmtlib header dependency, move this from the header to the .cpp side.

Reply:
There is no Color.cpp/Color8Bit.cpp file currently. To create one?

Reply:
Yes, create one.